pkg/scheduler/api/shared_device_pool.go

```diff
@@ -47,7 +47,7 @@ type Devices interface {
 	// Error is used for internal plugin errors, unexpected input, etc.
 
 	// 2: Unschedulable
-	// Unschedulable is used when a plugin finds a pod unschedulable. The scheduler might attempt to
```

Comment:
seems the previous description can explain its meaning.

Reply:
Yeah, after I read the comments, I really need to change them back. yes.